core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -66,15 +66,19 @@
   protected static final boolean unsafe = Boolean.parseBoolean(CarbonProperties.getInstance()
       .getProperty(CarbonCommonConstants.ENABLE_UNSAFE_COLUMN_PAGE,
           CarbonCommonConstants.ENABLE_UNSAFE_COLUMN_PAGE_DEFAULT));
+  // compressor to be used for this column
+  protected String columnCompressorName;
```

Comment:
ColumnPageEncoderMeta also has a compressor name variable.
Is it necessary to use re-factory it?

Reply:
refactored~